ChainReactionBindings/Bindings/ItemDetailsPageBindings.cs

```diff
@@ -141,19 +141,13 @@ public void ThenISeeSizeChartIsOpened()
 		[When(@"I scroll to element")]
 		public void ThenIScrollToElement()
 		{
-			var element = page.ContainerElement.FindElement(By.ClassName("crcPDPList"));
-			Actions act = new Actions(Browser.Driver);
-			act.MoveToElement(element);
-			act.Perform();
+			page.ScrollToElement();

```

Comment:
Rename ScrollToElement func and rename step to make it more readable

Reply:
done